docs/gha-workflows.md

```diff
@@ -145,3 +146,72 @@ jobs:
     with:
       github-run-id: "${{ github.event.workflow_run.id }}"
 ```
+# GitHub Actions: Integration Test Template Configuration
+
+This section details the inputs for the GitHub Actions workflow defined in `.github/workflows/test-integration-template.yaml` within the Camunda Platform Helm repository. These inputs allow you to customize Helm chart deployments for integration testing.
+
+## Workflow Inputs
+
+```yaml
+jobs:
+  ...
+  helm-deploy:
+    name: Helm chart Integration Tests
+    uses: camunda/camunda-platform-helm/.github/workflows/test-integration-template.yaml@main
+    secrets: inherit
+    with:
+      # Unique identifier used in the deployment hostname
+      # Required: true
+      identifier: 'dev-console-sm'
+
+      # Git reference for the Camunda Helm chart repository 
+      # Default: 'main'
+      # Required: false
+      camunda-helm-git-ref: 'main'
+
+      # Git reference of the caller's repository (branch, tag, or commit SHA) that initiated the workflow
+      # Default: 'main'
+      # Required: false
+      caller-git-ref: 'feature-branch'
+
+      # Whether to keep the test deployment after the workflow is completed
+      # Default: false
+      # Note: All persistent deployments will be deleted frequently to save costs
+      # Required: false
+      persistent: false
+
+      # Specifies the cloud platform that is currently used
+      # Default: 'gke'
+      # Required: false
+      platforms: 'gke'
+
+      # Types of operations to perform with the Helm chart, like install, upgrade
+      # Default: 'install'
+      # Required: false
+      flows: 'install'
+
+      # Flag to enable or disable the execution of test scenarios after Helm chart deployment
+      # Default: true
+      # Required: false
+      test-enabled: true
+
+      # Pass extra values to the Helm chart during deployment
+      # Required: false
+      extra-values: |
+        global:
+          image:
+            tag: 8.2.10
+        console:
+          image:
+            tag: xyz
+
+# Note:
```

Comment:
Let's add this outside the code part (just before `## Additional Information`)

Reply:
done